scenes/cslib_menu.txt

```diff
@@ -105,38 +107,54 @@
 *label build_simple_with_cancel
 *params
 *temp allow_cancel true
-*goto _build_simple_common
+*gosub _build_simple_common
+*gosub _render_choice
+*return
+
+
+*comment ::: INTERNAL FUNCTIONS BELOW (DO NOT CALL DIRECTLY!) :::
+*label _build_simple_common
+*gosub _init_state
+*temp prefix "param"
+*temp filter ""
+*temp filter_value ""
+*temp page_count "@{(param_count <= const_items_per_page) 1|${(param_count / const_items_per_page)}}"
```

Comment:
This stuff is getting really fancy - and it's really tricky for me to follow. Let's just test this stuff in and out.

Reply:
It's basically a ternary operator: page_count = (param_count <= page_length) ? 1 : (param_count / page_length).
Happy to change variable names if that would help.